docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
     command: sh -c "./wait-for.sh db:3306 -- npm run start:dev"
 
   db:
+    platform: linux/amd64
```

Comment:
I needed this on my Apple Silicon, but this is probably wrong for others. Anyone know if it's safe or what the proper fix should be?

Reply:
`docker-compose up --build` works on WSL2 on Windows with this property set.  `docker-compose` is version 2.29.2